pyproject.toml

```diff
@@ -46,7 +46,7 @@ python = "^3.7"
 numpy = "^1.21.0"
 scipy = "^1.4.1"
 pandas = "^1.1.0"
-openmdao = ">=3.10, <=3.21"
+openmdao = ">^3.10"
```

Comment:
```suggestion
openmdao = "^3.10"
```

Reply:
Thanks. Fixed in afbbd1eb.
The watchman tests now fail at unit tests, as expected 😄

Reply:
For the record, the typo didn't bothered `poetry lock --no-update` 😅

Reply:
Indeed 😄